.github/workflows/docker.yml

```diff
@@ -30,10 +36,15 @@ jobs:
         id: meta
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=raw,value=latest,enable=${{ github.event_name == 'release' }}
+            type=semver,pattern={{raw}},enable=${{ github.event_name == 'release' }}
+            type=raw,value=nightly-dev,enable=${{ github.ref == 'refs/heads/main' && github.event_name == 'push' || github.event_name == 'workflow_dispatch'}}
+            type=raw,value={{date 'YYYYMMDD-hhmmss' tz='America/New_York'}},enable=${{ github.ref == 'refs/heads/main' || github.event_name == 'workflow_dispatch' }}
```

Comment:
could we use ISO 8601 time format and UTC instead of Eastern?

Reply:
Changed to UTC but leaving the time format as usual. ISO8601 uses the colons which can't be used as a tag.